replit.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs }: {
+    deps = [
+        pkgs.yarn
+        pkgs.esbuild
+        pkgs.nodejs-16_x
```

Comment:
Can't we use nodejs 18?

Reply:
ReplIt does not support nodejs 18 @Nishith-Savla

Reply:
The only thing manually added here is `pkgs.openssl` since `prisma generate` was giving an OpenSSL error